paper.md

```diff
@@ -0,0 +1,178 @@
+Table 1: Agreement on statements by students from 2 workshops taught at the Netherlands eScience Center. 
+The results from these 2 workshops are a good representation of the general feedback we get when teaching this workshop.
+
+|                                                                           |          POOR    |     FAIR    |     GOOD    |     VERY GOOD    |     EXCELLENT    |     N/A    |     TOTAL    |     WEIGHTED AVERAGE    |
+|---------------------------------------------------------------------------|------------------|-------------|-------------|------------------|------------------|------------|--------------|-------------------------|
+|     Introduction into Deep Learning                                       | 0 (0%)           | 2 (5%)      | 10 (27%)    | 8 (22%)          | 17 (46%)         | 0 (0%)     | 37           | 4,1                     |
+|     Classification by a Neural Network using Keras (penguins dataset)     | 0 (0%)           | 1 (3%)      | 5 (13%)     | 16 (42%)         | 16 (42%)         | 0 (0%)     | 38           | 4,2                     |
+|     Monitoring and Troubleshooting the learning process (weather dataset) | 0 (0%)           | 0 (0%)      | 4 (11%)     | 18 (47%)         | 16 (42%)         | 0 (0%)     | 38           | 4,3                     |
+|     Advanced layer types (CIFAR-10 dataset)                               | 0 (0%)           | 2 (5%)      | 5 (13%)     | 7 (18%)          | 16 (42%)         | 8 (21%)    | 38           | 4,2                     |
+Table 2: Quality of the different episodes of the workshop as rated by students from 2 workshops taught at the Netherlands eScience Center. 
+The results from these 2 workshops are a good representation of the general feedback we get when teaching this workshop.
+
+# Citations
+
+Citations to entries in paper.bib should be in
+[rMarkdown](http://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html)
+format.
+
+For a quick reference, the following citation commands can be used:
+- `@author:2001`  ->  "Author et al. (2001)"
+- `[@author:2001]` -> "(Author et al., 2001)"
+- `[@author1:2001; @author2:2001]` -> "(Author1 et al., 2001; Author2 et al., 2002)"
+
+# Figures
+
+Figures can be included like this: ![Example figure.](figure.png)
+
+# Acknowledgements
+We would like to thank all instructors and helpers that taught the course,
+the community of people that left contributions to the project, no matter how big or small, 
+and the people around the world that piloted this workshop at their institutes.
+We thank the Carpentries for providing such a great framework for developing this lesson material.
+We thank all students enrolled in the workshops that were taught using this lesson material for providing us with feedback.
```

Comment:
Is it possible to name the Instructors who taught workshops and gave feedback? I am not sure of the conventions for Acknowledgements in JOSE, but I think it would be nice to name-check people if we know their names.

Reply:
Good point, done in d90cd93e1213e027f985b3b40e5700c2cf3d379b

Reply:
@qualiaMachine @NidhiGowdra @unode can you check if we wrote your names and affiliations correctly in https://github.com/carpentries-incubator/deep-learning-intro/commit/d90cd93e1213e027f985b3b40e5700c2cf3d379b?